whipper/common/mbngs.py

```diff
@@ -58,13 +58,13 @@ class TrackMetadata(object):
 
 class DiscMetadata(object):
     """
-    @param artist:       artist(s) name
-    @param sortName:     release artist sort name
-    @param release:      earliest release date, in YYYY-MM-DD
-    @type  release:      unicode
-    @param title:        title of the disc (with disambiguation)
-    @param releaseTitle: title of the release (without disambiguation)
-    @type  tracks:       C{list} of L{TrackMetadata}
+    :param artist:       artist(s) name
+    :param sortName:     release artist sort name
+    :param release:      earliest release date, in YYYY-MM-DD
+    :type  release:      unicode
+    :param title:        title of the disc (with disambiguation)
+    :param releaseTitle: title of the release (without disambiguation)
+    :type  tracks:       list of :any:`TrackMetadata`
```

Comment:
```suggestion
    :type  tracks:       :class:`list` of :any:`TrackMetadata`
```

Reply:
I think I'm going to use this:

 ```
:type  tracks:       list(:class:`TrackMetadata`)
```

Reply:
My suggestion was for an as 1:1 conversion as possible. Do you have a link to documentation for how `list(SomeClass)` is interpreted by Sphinx?

Reply:
https://www.pydev.org/manual_adv_type_hints.html

Reply:
If it's out of scope, I'll add these (and similar changes in #389) :smile:

Reply:
I wouldn’t say they’re out‐of‐scope for this PR—it’s still converting epydoc format to reST/Sphinx format.

Reply:
Also not sure what https://www.pydev.org/manual_adv_type_hints.html is, but it’s not upstream Sphinx docs at least. Trying to see if I can find this documented in Sphinx’s docs too.

Reply:
I asked about it on IRC in #sphinx-doc, but maybe just leave it out for now? If we either determine that there’s an official reST/Sphinx way we can do that, or we can make another PR to make it conform to the Eclipse way of doing things. Seems like it’s a reach for the scope of this PR which is *so close* now, so it also seems like this is such a small thing to hold the merging of the PR back on the basis of.